uavcan/node/port/7510.List.0.1.uavcan

```diff
@@ -0,0 +1,22 @@
+# A list of ports that this node is using:
+# - Subjects published by this node (whether periodically or ad-hoc).
+# - Subjects that this node is subscribed to (a datalogger or a debugger would typically subscribe to all subjects).
+# - RPC services consumed by this node (i.e., service clients).
+# - RPC services provided by this node (i.e., service servers).
+#
+# All nodes should implement this capability to provide network introspection and diagnostic capabilities.
+# This message should be published using the fixed subject-ID as follows:
+# - At the OPTIONAL priority level at least every MAX_PUBLICATION_PERIOD seconds.
+# - At the OPTIONAL or SLOW priority level within MAX_PUBLICATION_PERIOD after the port configuration is changed.
+#
+# pragma: no-bit-length-limit
```

Comment:
?

Reply:
Remember this quick-and-dirty `test.py` DSDL linter we use for validating the data type definitions? We have a tracking issue here documenting the need to create a *proper solution* instead of the current *quick hack*: #68.

The *hack* has this pragma to not fail the check if the data type is too large:

https://github.com/UAVCAN/public_regulated_data_types/blob/2eea7e6bbcc654b1516f6fb3435b8a2d44baaa1b/test.py#L113

If you think it causes confusion, we can easily just drop this check. Its value is rather low anyway.